qubesmanager/settings.py

```diff
@@ -993,13 +996,10 @@ def clone_vm(self):
 
     ######### advanced tab
 
-    def __init_advanced_tab__(self):
+    def __init_advanced_tab__(self):  # pylint: disable=too-many-statements
 
         vm_memory = getattr(self.vm, "memory", None)
         vm_maxmem = getattr(self.vm, "maxmem", None)
-        vm_preload_dispvm = int(
```

Comment:
I think the preload-disposables changes should all be in one commit (and the qrexec and  black stuff in their own commits)